component/site/controllers/icalevent.php

```diff
@@ -107,7 +111,7 @@ function detail() {
 		// View caching logic -- simple... are we logged in?
 		$cfg	 = JEVConfig::getInstance();
 		$joomlaconf = JFactory::getConfig();
-		$useCache = intval($cfg->get('com_cache', 0)) && $joomlaconf->get('caching', 1);
+		$useCache = intval($cfg->get('com_cache', 0)) && $joomlaconf->get('caching', 1) && $viewType != "pdf";
 		$user = JFactory::getUser();
```

Comment:
why not have a variable e.g. $useCache and change it in the plugin - then its not hard coded in here

Reply:
move it up